types/office-js/index.d.ts

```diff
@@ -15817,11 +15831,18 @@ declare namespace Office {
          *
          * **Errors**:
          *
-         * - `HTTPRequestFailure`: The request has failed. Please look at the diagnostics object for the HTTP error code.
+         * - `GenericTokenError: An internal error has occurred.` - In Exchange Online environments, this error occurs when the token can't be retrieved because legacy Exchange tokens
+         * for Outlook add-ins are turned off. We recommend using NAA as a single sign-on solution for your add-in. For guidance on how to implement NAA, see
+         * {@link https://learn.microsoft.com/office/dev/add-ins/develop/enable-nested-app-authentication-in-your-add-in | Enable SSO in an Office Add-in using nested app authentication}.
+         *
+         * - `HTTPRequestFailure: The request has failed. Please look at the diagnostics object for the HTTP error code`.
```

Comment:
I noticed some of these include the period within the backticks while others don't. Is it that the error message does/doesn't end with a period?

Reply:
The period should be within the backticks. Fixed the typo.